bin/update-ips.bash

```diff
@@ -71,14 +71,102 @@ diffDNSIPs() {
 
 # Updates the list from the file and yaml path specified with IPs fetched from the domain
 updateDNSIPs() {
-    read -r -a IPS <<< "$(getDNSIPs "${1}")"
+    endpoint="${1}"
+    inputSource="${2}"
+    inputConfig="$3"
 
-    yq4 -i "${2}"' = []' "${3}"
-    for ip in "${IPS[@]}"; do
-        yq4 -i "${2}"' |= . + ["'"${ip}"'/32"]' "${3}"
-    done
+    read -r -a IPS <<<"$(getDNSIPs "${endpoint}")"
+
+    IPS_copy=("${IPS[@]}")
+
+    configIPS=()
+    while IFS= read -r line; do
+        if [[ $line != */32 ]]; then
+            configIPS+=("${line//[- ]/}")
+        fi
+    done < <(yq4 e "$inputSource" "$inputConfig")
+
+    inside_subnet=()
+    working_subnet=()
+
+    # Clear config-values
+    yq4 -i "$inputSource"' = []' "$inputConfig"
+
+    # IF config-value is not empty, go ahread with the following:
+    # 1. Check if any got kubectlIPs fits inside of the IPs existing in the config-files.
+    # 2. IF True = Put the working subnet-address and kube-address in assigned arrays for comparison and remove copies.
+    # 3. Merge the working subnet-adresses and kube-addresses into a finalized list.
+    if [ "${configIPS[*]}" != '[]' ]; then
```

Comment:
Will this ever be `[]`? I think that it will just be `""` if the list is empty. And if you want to check if it's empty, you can do 

```suggestion
    if [ "${#configIPS[@]}" -gt 0 ]; then
```